.gitignore

```diff
@@ -1,4 +1,17 @@
 *.bak
+clab-sros-telemetry
 .venv/
+venv/
 .idea/
-*.code-workspace
+srl-sros-gnmi-config-lab/.config.clab.yml.bak
+srl-sros-gnmi-config-lab/clab-config
```

Comment:
these are not needed, global rules below take care of that iirc

Reply:
We should also put them into `.gitignore`